shared/test/test_net_sim_api.rb

```diff
@@ -0,0 +1,55 @@
+require 'minitest/autorun'
+require 'rack/test'
+require File.expand_path '../../../deployment', __FILE__
+require File.expand_path '../../middleware/net_sim_api', __FILE__
+
+ENV['RACK_ENV'] = 'test'
```

Comment:
does this make real api calls? and since it's not in the production env does it affect the production tables?